### ISSUE TITLE
Kit documentation needs updated?

### ISSUE THREAD

**Issue by jdeaton** [microsoft/vscode-cmake-tools#2761]
### Brief Issue Summary

The below excerpt from `src/kit.ts` seems to indicate there was a change in naming for the kit file. 

```
/**
 * The path to the user-local kits file.
 */
export const USER_KITS_FILEPATH = path.join(paths.dataDir, 'cmake-tools-kits.json');

/**
 * The old path where kits were stored. Upgraded in 1.1.3
 */
export const OLD_USER_KITS_FILEPATH = path.join(process.platform === 'win32' ? paths.roamingDataDir : paths.dataDir, 'cmake-tools.json');
```

The current `docs/kits.md` seems to reference the old naming. If this is indeed an error, I will do a pull request to fix it, but I wanted to make sure I was understanding correctly before doing so.

**Comment by elahehrashedi** [comment]
Thank you for pointing out this error in the documentation. The user-local kits' file name should be modified to `'cmake-tools-kits.json'`.
Please feel free to send a PR.

**Comment by jdeaton** [comment]
Fixed in #2772.
